src/main/kotlin/com/openlattice/linking/LinkingConfiguration.kt

```diff
@@ -43,6 +44,8 @@ private val DEFAULT_ENTITY_TYPES = setOf("general.person")
  */
 @ReloadableConfiguration(uri = "linking.yaml")
 data class LinkingConfiguration(
+        @JsonProperty("search") val searchConfiguration: SearchConfiguration,
```

Comment:
I think the properties should have the same name as in the yaml.

Reply:
Ah, copy-paste fail. Thanks!